### PR TITLE
Add : New resource in Icon subcategory

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
## Fixes Issue

Closes : #910 

## Screenshots

image

### CODE REVIEW

**Review by ujjawaltyagii — COMMENTED** [review]
It looks good to me ! @CBID2  @rupali-codes could you review my PR :)

**Comment by rupali-codes** [comment]
@ujjawaltyagii please resolve merge conflicts 😄

File: database/frontend/icons.json [hunk 1]
```diff
@@ -138,5 +138,12 @@
     "url": "https://www.iconfinder.com/",
     "category": "frontend",
     "subcategory": "icons"
+  },
+  {
+    "name": "Vertex",
+    "description": "The Vertex website offers a collection of freely available 3D icons that can be fully customized and utilized without any cost.",
```
Comment (CBID2) [inline database/frontend/icons.json:144]: [Proofreading]
```suggestion
    "description": "The Vertex website offers a collection of freely available 3D icons that can be fully customized and used for free.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just made some edits to the description @ujjawaltyagii.

**Comment by ujjawaltyagii** [comment]
Done with all the changes ! @rupali-codes @CBID2

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by ujjawaltyagii** [comment]
@rupali-codes could you review my PR

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**